packages/common/tsconfig.browser.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@ethereumjs/config-typescript/tsconfig.browser.json",
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "src/**/*.json"],
```

Comment:
These `jsons` were accidentally missing from the tsc browser config. 

The errors looked like:
```
20 10 2020 15:04:25.170:ERROR [framework.browserify]: bundle error
20 10 2020 15:04:25.172:ERROR [framework.browserify]: Error: Can't walk dependency graph: Cannot find 
module './mainnet.json' from '/ethereumjs-vm/packages/common/dist.browser/chains/index.js'
    required by /ethereumjs-vm/packages/common/dist.browser/chains/index.js
``` 
...and the stack trace shows karma pulling files from `dist.browser` so that's 👍 .